src/components/commentsList/Comment.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+
+export const Comment: FC<Comment> = ({ name, body, email }) => (
```

Comment:
Принимай здесь объект `comment`, а из него уже доставай все остальное
```suggestion
export const Comment: FC<Props> = ({ comment }) => (
```